projectTemplates/ts-server/dockerfile

```diff
@@ -5,13 +5,18 @@ RUN npm install
 COPY . .
 RUN npm run build
 
-
-FROM node:12.18.3-slim as production
+FROM node:12.18.3-slim as node-base
```

Comment:
any reason why not alpine?

Reply:
npm install with our package.lock fails on alpine.